DataDog/datadog-serverless-functions
# Forwarder: Additional target lambdas

### What does this PR do?



This PR adds an optional parameter in the cloudformation stack to have an additional lambda that gets called with the same event payload the forwarder receives. This can allow customers to ship logs to other vendors such as Logstash (ELK). The parameter is `additionalLambdaTargetARNs`.

### Motivation



This is always a blocker for customers that want to use DD instrumentation, but also ships to another log vendor.

### Testing Guidelines



I **extended** the existing integration tests to deploy two new lambdas. After calling the normal flow of integration tests I then verify that the two lambdas were called with the same data as the forwarder.

Additionally,  I was able to test the Cloudformation template **manually** (via upload) - then updating the forwarder lambda code to my latest code. After that initial set up, I was able to add a lambda to to get instrumented (via the plugin with the new forwarder ARN) After calling the test lambda I verified that the logs were being dual shipped to both the forwarder and the `additionalLambdaTargets` (2 other lambdas in this case).

### Additional Notes



### Types of changes


### Check all that apply
